staging/src/k8s.io/csi-translation-lib/plugins/azure_disk.go

```diff
@@ -127,9 +128,7 @@ func (t *azureDiskCSITranslator) TranslateInTreeInlineVolumeToCSI(volume *v1.Vol
 		pv.Spec.PersistentVolumeSource.CSI.FSType = *azureSource.FSType
 		pv.Spec.PersistentVolumeSource.CSI.VolumeAttributes[azureDiskFSType] = *azureSource.FSType
 	}
-	if azureSource.Kind != nil {
-		pv.Spec.PersistentVolumeSource.CSI.VolumeAttributes[azureDiskKind] = string(*azureSource.Kind)
-	}
+	pv.Spec.PersistentVolumeSource.CSI.VolumeAttributes[azureDiskKind] = managed
```

Comment:
Can something not managed be specified?

Reply:
since CSI driver only supports managed, we can only support `managed` in csi migration.

Reply:
> Can we make the CSI driver case insensitive?

In theory, we could fix it in volume validation, while since this issue only exists in csi migration, may be not necessary now

Reply:
What happens if someone is using a volume that is unmanaged with CSI migration?

Reply:
> What happens if someone is using a volume that is unmanaged with CSI migration?

it's not supported, volume would be in unattached state forever. unmanaged disk is actually deprecated in Azure, if user still wants to use unmanaged disk, they need to turn off csi migration for disk.

Reply:
Can a user use unmanaged disks with in-tree driver today? If so, we may need to also send a deprecation notice for this.

Also, should we return an error if the kind is not supported?

Reply:
already added the error code and depreciation notice in doc notes if kind is not supported.

Reply:
Thanks, can you also clarify, is there any documentation we can point to users on how to migrate their disks from unmanaged to managed?